### PR TITLE
[service-hero] Add hero images to services list

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

bunny

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/909lULy0/65-i-want-small-versions-of-the-hero-images-above-each-service-title-on-the-services-list-page

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

(also updated the home page links to point to individual service pages)

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

main feature:
- go to "services" in the top navigaton
- do the pages have hero images?

home page title links
- top of the home page
- Custom Web App *Design*, *Development*, & *Training* now link to individual pages

## Show me
_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[I want small versions of the hero images above each service title on the services list page.](https://trello.com/c/909lULy0/65-i-want-small-versions-of-the-hero-images-above-each-service-title-on-the-services-list-page)

File: content/_includes/post.macros.njk [hunk 1]
```diff
@@ -173,6 +173,40 @@ params:
   </article>
 {% endmacro %}
 
+{# @docs
+label: grid
+category: posts
+note: A simple grid of pages, without full "card" reource styling
```
Comment (stacyk) [inline content/_includes/post.macros.njk:179]: "resource" typo

File: content/_includes/post.macros.njk [hunk 2]
```diff
@@ -173,6 +173,40 @@ params:
...
+    type: number (1-6)
+    default: 3
+    note: Heading level for page titles
+  collections:
+    type: 11ty collections object
+    default: none
+#}
+{% macro grid(
+  pages,
+  level=3,
```
Comment (stacyk) [inline content/_includes/post.macros.njk:193]: I love this heading level option!

File: content/services/planning.md [hunk 3]
```diff
@@ -157,11 +157,10 @@ action:
...
   to make sure we thoroughly understand your project.
-  We'll help you pinpoint the right
-  digital product -- a combination of customer/user needs and business
-  goals -- with a robust plan for development and launch. We'll create an
-  interactive wireframe to visualize your application or website, and a
-  prioritized estimate to keep you in control of cost.
+  We'll help you pinpoint the right digital product -- 
+  with interactive wireframes,
+  prioritized estimates,
+  and a robust plan for development and launch,
```
Comment (stacyk) [inline content/services/planning.md:163]: The comma here should be a period I believe.

**Pull request merged**